src/data/weapons/assaultRifles.js

```diff
@@ -11,7 +11,7 @@ const base = [
 	'Kastov 545',
 ]
 
-const dlc = []
+const dlc = ['M13B', 'Chimera']
```

Comment:
```suggestion
const dlc = ['M13B', 'Chimera']

const comingSoon['M13B', 'Chimera']
```

And then add a property below on the weapon object:

```javascript
comingSoon: comingSoon.includes(weapon)
```